hopsworks-api/src/main/java/io/hops/hopsworks/api/project/MetadataService.java

```diff
@@ -585,50 +586,46 @@ private Response mutateMetadata(Users user, String metaObj, MetadataOp op)
             build();
 
   }
-
-  @POST
-  @Path("attachSchemaless")
-  @Consumes(MediaType.APPLICATION_JSON)
-  @AllowedProjectRoles({AllowedProjectRoles.DATA_SCIENTIST, AllowedProjectRoles.DATA_OWNER})
-  public Response attachSchemalessMetadata(String metaObj) throws DatasetException, MetadataException {
-    processSchemalessMetadata(metaObj, false);
-    return noCacheResponse.getNoCacheResponseBuilder(Response.Status.ACCEPTED).build();
-  }
-
-  @POST
-  @Path("detachSchemaless")
+  
+  @PUT
+  @Path("schemaless/{xattrName}/{path: .+}")
```

Comment:
@maismail Please add Swagger ApiOperation annotations to the endpoints. An example is [this](https://github.com/logicalclocks/hopsworks/blob/master/hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobsResource.java#L145).

Reply:
done